packages/esm-api/src/breadcrumbs/types.ts

```diff
@@ -0,0 +1,32 @@
+export interface BreadcrumbSettings {
+  /**
+   * Gets the path of breadcrumb for navigation purposes.
+   */
+  path: string;
+  /**
+   * Gets a custom matching function to know if the breadcrumb should be selected.
+   *
+   * In case of a missing matcher it uses the path to regexp result of the given path.
+   *
+   * In case of a string it uses the path to regexp result of the given matcher.
```

Comment:
This could be clarified a bit.

```suggestion
   * A string or RegEx that determines whether the breadcrumb should be displayed.
   * It is tested against the current location's path.
   * 
   * If `matcher` is a string, it can contain route parameters. e.g. `/foo/:bar`.
   *
   * Can be omitted; the value of `path` is used as the default value.
```